build/stage-build.yml

```diff
@@ -13,22 +13,22 @@
     configFilePath: build/gitversion.yml
   displayName: 'Calculate version'
 
-- task: NuGetToolInstaller@1
```

Comment:
@MatFillion Is this something we want to keep? (using MSBuild@1 to restore the packages)

Reply:
This was added because we were facing an issue with the classic nuget restore task (I don't remember exaclty which one, but I can search if you want to track it). We can keep msbuild in my opinion.